lib/components/AlbumScreen/song_menu.dart

```diff
@@ -39,8 +40,8 @@ import 'download_dialog.dart';
 Future<void> showModalSongMenu({
   required BuildContext context,
   required BaseItemDto item,
-  ColorScheme? playerScreenTheme,
   bool showPlaybackControls = false,
+  bool usePlayerTheme = false,
```

Comment:
Can we rename this to what it actually does (not change theme if the current track changes)? Maybe something like `staticTheme`.  
And is there any scenario where we *would* want the menu theme to automatically update if the current track changes?

Reply:
There aren't any situations we want the theme to change, because the menu content never does.  The theme should always be static, but sometimes the caller supplies it and sometimes we want to use the item that's playing when the menu gets opened, which is what this flag flips between.